### DISCUSSION TITLE
non repeating keys

### DISCUSSION THREAD

**Discussion by klue-less** [isontheline/pro.webssh.net#358] [Ideas]
'Bought Pro version yesterday.  Tried it out with two encrypted private keys with SSH to a custom port number and SFTP to the standard port.  Changed the terminal to XTERM-256COLOR, and that works too, but once connected, there something is missing:  If soft keyboard keys are held down, they do not repeat as a real keyboard would.

Please add repeating keys when they are held down.

**Comment by isontheline** [comment]
Hey @klue-less 👋

Thanks for the idea!

I have created an issue to track progress : https://github.com/isontheline/pro.webssh.net/issues/361
